prow/scripts/cluster-integration/kyma-gke-upgrade.sh

```diff
@@ -429,7 +429,7 @@ function upgradeKyma() {
 function testKyma() {
     shout "Test Kyma"
     date
-    "${TEST_INFRA_SOURCES_DIR}"/prow/scripts/kyma-testing.sh
+    "${KYMA_SCRIPTS_DIR}"/testing-upgrade.sh
```

Comment:
Does this scripts supports JUnit generation? ` "${TEST_INFRA_SOURCES_DIR}"/prow/scripts/kyma-testing.sh` is using Kyma CLI that generates it

Reply:
JUnit reports are used for TestGrid dashboards: https://testgrid.k8s.io/kyma-postsubmit#post-master-kyma-gke-upgrade